### ISSUE TITLE
Pig support

### ISSUE THREAD

**Issue by leijurv** [cabaletta/baritone#223]
Riding a pig should work but it currently doesn't

Baritone should detect you're on a pig and if so:

Change the way it sets yaw so it works while riding an entity (right now it doesn't actually control which way the pig or the player are facing)

Disable allowSprint
Disable allowParkour
Disable allowPlace (pig's in the way lol)
Disable allowParkourPlace
Disable allowBreak (carrot on a stick + toolset autotool = not work)
Disable allowWaterBucketFall
Disable allowWalkOnBottomSlab (it's glitchy on a pig)

**Comment by leijurv** [comment]
https://youtu.be/JqRN3Yin-po

Initial control added here https://github.com/cabaletta/baritone/commit/314f2804005785eab4d4685966cc8e4efe11c53c, still need to disable all that crap listed above. Also need to make it avoid water entirely.

**Comment by leijurv** [comment]
Also need some way to disable walking on water in water (floating), maybe another flag alongside canwalkonwater which is like canfloatinwater idk

**Comment by leijurv** [comment]
yeah so it can control pigs, right, but we still gotta disable these settings so it doesn't do impossible things to the pig

**Comment by ImUrX** [comment]
no foreseeable progress on this?

dont tempt me up please
